pkg/api/v1alpha1/cluster_types.go

```diff
@@ -23,6 +23,9 @@ const (
 	// managementAnnotation points to the name of a management cluster
 	// cluster object
 	managementAnnotation = "anywhere.eks.amazonaws.com/managed-by"
+
+	// defaultEksaNamespace is the default namespace for EKS-A resources when not specified.
+	defaultEksaNamespace = "default"
```

Comment:
shouldn't this be `eksa-system`? I think I'm missing something here :)

Reply:
so this is the default namespace for eksa resources (cluster, datacenter, machineconfig, gitopsconfig etc)